### PR TITLE
Implement -p flag

### PR DESCRIPTION
**Pull request by sosedoff**
This PR adds support for `-p` flag so that documentation is correct.

### CODE REVIEW

File: shelf.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 )
 
 var storageDir = flag.String("s", "", "Path to store files, instead of Postgres")
+var listenPort = flag.String("p", "", "Port to listen on")
```
Comment (titanous) [inline shelf.go]: The default should be `"3001"`.

Comment (sosedoff) [inline shelf.go]: Ok, will fix

**Comment by titanous** [comment]
Thanks! The commit needs a [DCO sign-off](https://flynn.io/docs/contributing#developers-certificate-of-origin).

**Comment by sosedoff** [comment]
Should be fine now

**Comment by Qard** [comment]
Just a thought; it's a bit weird that the PORT env variable is prioritized over the flag. Might cause some unexpected behaviour when that env variable is set for something else.

**Comment by titanous** [comment]
@Qard yep, that's true. Feel free to submit a PR that changes the behavior.

**Pull request merged**